modules/engage-theodul-plugin-video-videojs/src/main/resources/static/main.js

```diff
@@ -419,20 +420,26 @@ define(['require', 'jquery', 'underscore', 'backbone', 'basil', 'bowser', 'engag
     if (filterFormats == undefined) {
       return tracks;
     }
-    var filterFormatsArray = filterFormats.split(',');
-    var newTracksArray = [];
+    var newTracksArray = [],
+        filterFormatsArray = (Array.isArray(filterFormats) ? filterFormats : filterFormats.split(','))
```

Comment:
Shouldn't `filterFormats` always be an array?

Reply:
Yes, probably the same as filterTags in filterTracksByTag, just that this code refers to splitting up a string so it might be because of the previous way that the configuration element was defined (comma separated string) instead of a list as in the new configuration file (etc/ui-config/mh_default_org/theodul/config.yml)

Reply:
Yes, and in the new pull request will handle it appropriately.